netzob/src/netzob/Inference/Grammar/EquivalenceOracles/AbstractEquivalenceOracle.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+#+---------------------------------------------------------------------------+
+#|          01001110 01100101 01110100 01111010 01101111 01100010            |
+#|                                                                           |
+#|               Netzob : Inferring communication protocols                  |
+#+---------------------------------------------------------------------------+
+#| Copyright (C) 2011-2017 Georges Bossert and Frédéric Guihéry              |
```

Comment:
This is an obsolete class not in the current 'next' branch of netzob. I guess you have merged your work with the 'master' branch or an old 'next' branch version. I would suggest to synchronize your PR branch with the current version of the 'next' branch.